scripts/pi-hole/js/index.js

```diff
@@ -420,6 +420,46 @@ $(function () {
   // Pull in data via AJAX
   updateSummaryData();
 
+  const zoomPlugin = {
+    /* Allow zooming only on the y axis */
+    zoom: {
+      wheel: {
+        enabled: true,
+        modifierKey: "ctrl" /* Modifier key required for zooming via mouse wheel */,
+      },
+      pinch: {
+        enabled: true,
+      },
+      mode: "y",
+      onZoom: function ({ chart }) {
+        // The first time the chart is zoomed, save the absolute maximum value
+        if (!chart.absMax) chart.absMax = chart.scales.y.max;
```

Comment:
When I zoom in the zoom speed seems to be normal, but when I try to zoom out the scale changes slowly, in very small steps:

![slow_zoom_out](https://github.com/pi-hole/web/assets/1385443/412697ad-7c96-4baa-a9ed-0ca2a1755a67)

Changing the code to use the original max bound (70 in my example) seems to fix this:

```suggestion
        // The first time the chart is zoomed, save the maximum initial scale bound
        if (!chart.absMax) chart.absMax = chart.getInitialScaleBounds().y.max;
```

Reply:
I did not know such a function exists